### ISSUE TITLE
See someone's followers (including your own)

### ISSUE THREAD

**Issue by mikedilger** [mikedilger/gossip#111]
filter [kind=3, #p=pubkey]
multiple relays
merge

**Comment by mikedilger** [comment]
This is a lot of data to load, and there is no way to get it all since people don't all use the same relays and you would need to get data from people you've never heard of before... since nostr isn't centralized, the best you can do is ask lots of relays for all their kind-3 events that reference you (or the person in question), and eliminate duplicates.  This can take several minutes, which is a UI problem.

And if you just want the count, HLL may be coming soon.

Luckily nostr doesn't have to be just one giant app.  We can instead put a link to some other app which does this, instead of doing it ourselves... like "view your followers" that opens some web-page nostr app that does this.

**Comment by mikedilger** [comment]
I decided to go ahead and try this anyhow.  I've got it all but the UI part which is the trickiest part because if you have somebody  with 6,000 followers (not uncommon at all) you cannot reasonably list that many *alphabatized* since you only have a set of 6,000 pubkeys and no way to sort them *alphabatized* on person name.  Holding 6000 person records is too big.  There *is* a way to only render the ones that are actually on screen, just no way to sort them.
